docs/concepts/merge.md

```diff
@@ -12,14 +12,14 @@ The Merge was executed on **September 15, 2022**.
 
 :::
 
-[The Merge](https://ethereum.org/en/upgrades/merge/) was an Ethereum upgrade that merged the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/) into Ethereum Mainnet, turning Mainnet into a combination of an [execution layer and consensus layer](#execution-and-consensus-clients). The Merge transitioned Mainnet from proof of work to [proof of stake consensus](proof-of-stake.md).
+[The Merge](https://ethereum.org/en/upgrades/merge/) was an Ethereum upgrade that merged the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/) into Ethereum Mainnet, turning Mainnet into a combination of an [execution layer and consensus layer](#execution-and-consensus-clients). The Merge transitioned Mainnet from proof of work to [proof-of-stake consensus](proof-of-stake.md).
 
 You can run Teku as a consensus client with:
 
-- Any execution client on Mainnet.
-- Any execution client on a testnet.
-- Besu on Mainnet.
-- Besu on a testnet.
+- Any execution client on Mainnet
+- Any execution client on a testnet
+- Besu on Mainnet
+- Besu on a testnet
```

Comment:
```suggestion
- Any execution client on Mainnet.
- Any execution client on a testnet.
- Besu on Mainnet.
- Besu on a testnet.
```

Please review the list guidelines from the Microsoft Style Guide: https://learn.microsoft.com/en-us/style-guide/scannable-content/lists. If items complete a sentence, they should end with a period.